src/Helper/Processor.php

```diff
@@ -36,6 +36,7 @@ public function render(SlidingPaginationInterface $pagination, array $queryParam
 
         $data['route'] = $pagination->getRoute();
         $data['query'] = \array_merge($pagination->getParams(), $queryParams);
+        $data['paginatorOptions'] = $pagination->getPaginatorOptions();
```

Comment:
let's go with "option" as name, it's simpler

Reply:
Changed as requested